src/index.js

```diff
@@ -1,123 +1,10 @@
-/** Copyright (c) 2018 Uber Technologies, Inc.
```

Comment:
Should probably keep this header. It is missing from some other files in this repo though, so we could do an audit later.